optimize/backend/src/test/java/io/camunda/optimize/service/ProcessOverviewServiceTest.java

```diff
@@ -0,0 +1,194 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.optimize.service;
+
+import static io.camunda.optimize.dto.optimize.DefinitionType.PROCESS;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.AssertionsForClassTypes.catchThrowable;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import io.camunda.optimize.dto.optimize.IdentityDto;
+import io.camunda.optimize.dto.optimize.IdentityType;
+import io.camunda.optimize.dto.optimize.UserDto;
+import io.camunda.optimize.dto.optimize.query.definition.DefinitionWithTenantIdsDto;
+import io.camunda.optimize.dto.optimize.query.processoverview.ProcessDigestDto;
+import io.camunda.optimize.dto.optimize.query.processoverview.ProcessDigestRequestDto;
+import io.camunda.optimize.dto.optimize.query.processoverview.ProcessOverviewDto;
+import io.camunda.optimize.dto.optimize.query.processoverview.ProcessOverviewResponseDto;
+import io.camunda.optimize.dto.optimize.query.processoverview.ProcessUpdateDto;
+import io.camunda.optimize.service.db.reader.ProcessOverviewReader;
+import io.camunda.optimize.service.db.writer.ProcessOverviewWriter;
+import io.camunda.optimize.service.digest.DigestService;
+import io.camunda.optimize.service.identity.AbstractIdentityService;
+import io.camunda.optimize.service.security.util.definition.DataSourceDefinitionAuthorizationService;
+import jakarta.ws.rs.ForbiddenException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.Set;
+import org.junit.jupiter.api.Test;
+
+public class ProcessOverviewServiceTest {
+
+  private final DefinitionService definitionService = mock(DefinitionService.class);
+  private final DataSourceDefinitionAuthorizationService definitionAuthorizationService =
+      mock(DataSourceDefinitionAuthorizationService.class);
+  private final ProcessOverviewWriter processOverviewWriter = mock(ProcessOverviewWriter.class);
+  private final ProcessOverviewReader processOverviewReader = mock(ProcessOverviewReader.class);
+  private final AbstractIdentityService identityService = mock(AbstractIdentityService.class);
+  private final KpiService kpiService = mock(KpiService.class);
+  private final DigestService digestService = mock(DigestService.class);
+
+  private final ProcessOverviewService processOverviewService =
```

Comment:
```suggestion
  private final ProcessOverviewService underTest =
```

I think we generally call it this for convention